system/Commands/ListCommands.php

```diff
@@ -73,7 +73,7 @@ class ListCommands extends BaseCommand
 	 *
 	 * @var string
 	 */
-	protected $description = 'Lists the available commands.';
+	protected $description = 'List of available commands.';
```

Comment:
I think the former is more apt as it describes the command's function, i.e. what it is doing.